package.json

```diff
@@ -2,18 +2,27 @@
   "name": "picocolors",
   "version": "1.0.0",
   "main": "./picocolors.js",
+  "module": "./picocolors.mjs",
   "types": "./picocolors.d.ts",
   "browser": {
     "./picocolors.js": "./picocolors.browser.js"
   },
+  "exports": {
+    ".": {
+      "types": "./picocolors.d.ts",
+      "require": "./picocolors.js",
+      "import": "./picocolors.mjs"
+    }
+  },
   "sideEffects": false,
   "description": "The tiniest and the fastest library for terminal output formatting with ANSI colors",
   "scripts": {
+    "build": "node build-esm.mjs && tsc --module esnext tests/types.mts",
     "test": "node tests/test.js"
```

Comment:
Would it not be more logical to put the compiling of the types tests during the *test* script? As it is a way to tests that types are correct. 

Also maybe adding `--noEmit` option to not produce any compiled files of the test, no ?